packages/menu/package.json

```
@@ -21,6 +21,7 @@
     "@leafygreen-ui/lib": "^4.4.1",
     "@leafygreen-ui/popover": "^5.0.2",
     "@leafygreen-ui/palette": "^2.0.1",
+    "@leafygreen-ui/box": "1.1.1",
```

Let's always default to ranges so we can enable better hoisting.

```suggestion
    "@leafygreen-ui/box": "^1.1.1",
```

---

👍